### DISCUSSION TITLE
Hide 'Upload media' and 'My media' from left nav if user is not advancedUser

### DISCUSSION THREAD

**Discussion by petr0ff** [mediacms-io/mediacms#508] [Q&A]
**Describe the feature you'd like**
There is a setting CAN_ADD_MEDIA is settings.py. If I set it to "advancedUser" then user who is not advanced cannot upload media - it's ok, it works on permissions level. My suggestion is to hide both 'Upload media' and 'My media' for all users who have not "Advanced user" checked in django admin settings and if CAN_ADD_MEDIA="advancedUser".

**Describe alternatives you've considered**

**Additional context**

**Comment by mabruzzo3169** [comment]
Has this ever been sorted?
